website/source/docs/provisioners/ansible-local.html.markdown

```diff
@@ -41,15 +41,16 @@ Optional:
 * `extra_arguments` (array of strings) - An array of extra arguments to pass to the
   ansible command. By default, this is empty.
 
-* `inventory_groups` (string) - You can let Packer generate a temporary inventory
-  for you. It will contains only `127.0.0.1`. Thanks to `inventory_groups`,
-  packer will set the current machine into different groups and will
-  generate an inventory like:
+* `inventory_groups` (string) -  A comma-separated list of groups to which
+  packer will assign the host `127.0.0.1`. A value of `my_group_1,my_group_2`
+  will generate an Ansible inventory like:
 
+```text
   [my_group_1]
   127.0.0.1
   [my_group_2]
   127.0.0.1
+```
```

Comment:
Thanks for the update! Actually the whole thing should be indented so it's part of the list item. I'll tweak it.

E.g.

  * this is the list item

  ```
  and this is the example
  ```

  * and this is the next item

vs. 

  * this is the list item

```
and this is the example
```

  * and this is the next item